python/test/gluon/test_frontend.py

```diff
@@ -1315,3 +1317,34 @@ def test_auto_layout_broadcast():
     # CHECK: [[XBCAST2:%.*]] = tt.broadcast [[XCVT2]]
     # CHECK: arith.muli [[YBCAST2]], [[XBCAST2]] : tensor<16x16xi32, [[BLOCKED]]>
     _ = y * x
+
+
+@gluon.jit
+def amd_mfma_layout_kernel():
+    mfma_layout: ttgl.constexpr = cdna3.AMDMFMALayout(version=4, instr_shape=[32, 32], transposed=True,
```

Comment:
It's a bit strange to see `cdna3.AMDMFMALayout` with a `version=4` given that means cnda4.. I guess we should move the `AMDMFMALayout` to be under `amd/_layouts.py` and import into `amd/__init__.py` directly.